bin/check-redis-memory-percentage.rb

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env ruby
+#
+# Checks Redis INFO stats and limits values
+# ===
+#
+# Copyright (c) 2012, Panagiotis Papadomitsos <pj@ezgr.net>
+#
+# Released under the same terms as Sensu (the MIT license); see LICENSE
+# for details.
+
```

Comment:
can you add the standard header per http://sensu-plugins.io/docs/developer_guidelines.html#coding-style